crates/cli/Cargo.toml

```diff
@@ -25,4 +25,4 @@ clap = { version = "4.4", features = ["derive"] }
 env_logger = "0.10"
 sha2 = "0.10"
 
-schematools = { path = "../schematools" }
+schematools = { version = "0.17", path = "../schematools" }
```

Comment:
Note this isnt going to work perfectly with the automatic release versioning.

I believe it should be fine for most cases, as this sets the lower compatible version, but it will require manually bumping for any major release.  We can probably organise some automated changing of this version also.

Reply:
You can change the line to (full semversion):

```
schematools = { version = "0.17.1", path = "../schematools" }
```

You can add to https://github.com/kstasik/schema-tools/blob/master/.cz.toml#L6C50-L6C50:

```
    "crates/cli/Cargo.toml:^schematools",
```

If i remember correctly it uses regexp to find proper line and replaces old sem version to new one.

Reply:
https://commitizen-tools.github.io/commitizen/bump/#version_files

Reply:
I've installed cz and confirmed that worked.